__tests__/getPlacePhoto.test.ts

```diff
@@ -0,0 +1,37 @@
+import { Request, Response } from "express";
+import axios from "axios";
+import { getPlacePhoto } from "../src/rest/getPlacePhoto";
+import StatusCode from "../src/rest/status";
+
+jest.mock("axios");
+//eslint-disable-next-line @typescript-eslint/unbound-method
+const mockGet = axios.get as jest.MockedFunction<typeof axios.get>;
+
+const mockData = "Well hello there,";
+
+describe("Get place photo endpoint tests", () => {
+	it("returns an OK response if there is no error", async () => {
+		mockGet.mockResolvedValue({ data: mockData });
+
+		const mockSet = jest.fn();
+		const mockSend = jest.fn();
+		const mockStatus = jest.fn().mockReturnValue({ send: mockSend });
+
+		// getPlacePhoto needs an await to finish executing the promise...
+		// yet it does not return one so its not technically "awaitable"
```

Comment:
you don't need a return to be awaitable - if it returns a promise it's awaitable, or if it's async and has no return then it actually behind the scenes does return a Promise with a void resolution

Reply:
oh that's actually an issue with getPlacePhoto - it should have a return type of `Promise<void>` instead of `void`

Reply:
and i think it would be best practice to make it async and `await` the axios.get chain, because technically the function shouldn't end execution until a response is given

Reply:
>  i think it would be best practice to make it async and await the axios.get chain

agreed, fixed in my latest commit.